tests/unit/test_release_study.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import responses
+
+from assembly_uploader.release_study import StudyReleaseError, release_study
+from assembly_uploader.webin_utils import (
+    ENA_WEBIN,
+    ENA_WEBIN_PASSWORD,
+    ensure_webin_credentials_exist,
+)
+
+
+def test_release_study(study_submission_xml_dir, monkeypatch, tmp_path):
+    ena_dropbox = responses.add(
+        responses.POST,
+        "https://wwwdev.ebi.ac.uk/ena/submit/drop-box/submit",
+        body="""
+        This is a long receipt from the dropbox.
+        success="true"
+        Your study was released.
+        """,
+    )
```

Comment:
I couldn't find an example of this... but guessed it probably has `success="true"` in it like an `ADD` action.